src/api/docs/content/specs/config.yaml

```diff
@@ -239,6 +239,8 @@ components:
                       type: integer
                     optimizer:
                       type: integer
+                    upstreamTTL:
```

Comment:
```suggestion
                    upstreamBlockedTTL:
```

Reply:
done in the latest commit, needed to be done in 12 locations